functests/1_performance/performance.go

```diff
@@ -340,8 +342,134 @@ var _ = Describe("[rfe_id:27368][performance]", func() {
 			Expect(err).ToNot(HaveOccurred(), fmt.Sprintf("cannot find MachineConfig object %s", initialKey.Name))
 		})
 	})
+
+	Context("Verify API Conversions", func() {
+		It("[test_id:35887] Verifies v1 <-> v1alpha1 conversions", func() {
+
+			By("Checking v1 -> v1alpha1 conversion")
+			v1alpha1Profile := &performancev1alpha1.PerformanceProfile{}
+			key := types.NamespacedName{
+				Name:      profile.Name,
+				Namespace: profile.Namespace,
+			}
+
+			err := testclient.GetWithRetry(context.TODO(), key, v1alpha1Profile)
+			ExpectWithOffset(1, err).ToNot(HaveOccurred(), "Failed getting v1alpha1Profile")
```

Comment:
why not a plain Expect() ?

Reply:
I saw in an example earlier in the code, I will update.

Reply:
done